README.md

```diff
@@ -1,39 +1,74 @@
 # TikTokCommentsFetcher
-Script that fetches comments from a TikTok post 
+Script to parse comments from TikTok
 
-## Документация
+_Requires authentication via Google Account._
 
-### Требования:
-1) Проставить значения переменных окружения EMAIL, PASSWORD и COMMENTS_LIMIT.
-EMAIL и PASSWORD - данные для входа в почту gmail (комментарии нельзя смотреть без входа в аккаунт).
-COMMENTS_LIMIT - ограничение на количество комментариев (если пост очень популярный, все комментарии могут грузиться долго).
-2) Запускать лучше на Linux или MacOS, на Windows может сходу не работать, надо будет немного подправить скрипт.
+If you have problems, head to **Problems** section and to Issues.
 
-### Вход:
-Ссылка на пост.
+### How to use
+0. **Prerequisites.** This script works on Linux and Mac systems. Good work is not guaranteed for Windows. Create python virtual environment and install dependencies.
+```bash
+python3 -m venv/
+source venv/bin/activate
+python3 -m pip install -r requirements.txt
+```
+1. Open terminal and set environment variables EMAIL, PASSWORD, COMMENTS_LIMIT with gmail account email and password.
+> COMMENTS_LIMIT is used to limit comment parsing for posts with lots of comments. For example, if COMMENTS_LIMIT = 100, than script will parse first 100 comments and then stop. If COMMENTS_LIMIT is too big, script will work for too long.
 
-### Выход:
-CSV файл со столбцами: id, ник, ссылка на профиль, текст комментария. Дополнительно можно выводить, например, количество лайков комментария и ответы на него.
+```bash
+export EMAIL=<your_email@gmail.com> && \
+export PASSWORD=<password_for_google_account> && \
+export COMMENTS_LIMIT=<comment_limit>
+```
+2. Copy-paste link to video you want to parse into the `input.txt`.
+```
+echo "https://www.tiktok.com/?lang=en&is_copy_url=1&is_from_webapp=v3&item_id=6906436719541767425#/@youneszarou/video/6906436719541767425" >input.txt
+```
+3. Run python script `tiktok.py`.
+```
+python3 tiktok.py <input.txt
+```
 
-### Скорость парсинга
-Скорость скачивания можно оценить так:
-100 комментов - моментально.
-2000 комментов - минута.
-5к комментов - 10 минут.
 
-Скорость упирается не в язык программирования, технологии и подход к парсингу, а в ограничения веб версии тиктока. Тикток напрямую борется с выгрузкой комментариев, поэтому:
-1) они подгружаются динамически, по 20 штук
-2) следующие можно подгрузить только когда уже погрузил предыдущие. Поэтому нельзя распараллелить выгрузку в пределах одного поста.
-3) После скачивания 1000 комментариев тикток даёт следующие 20 примерно с задержкой в 1 секунду. Дальше задержка становится 5 секунд и тд
+### Result
+CSV file with columns: `id`, `nickname`, `link_to_profile`, `text`, `like_count`. No headers!
```

Comment:
Better insert example table. Also, in there person will see that there are no headers